coral/types/api.d.ts

```diff
@@ -2748,6 +2757,7 @@ export type operations = {
         operationType?: "CREATE" | "UPDATE" | "PROMOTE" | "CLAIM" | "DELETE";
         env?: string;
         search?: string;
+        order?: "OLDEST_FIRST" | "NEWEST_FIRST";
```

Comment:
how about using ascending, descending ?

Reply:
So I agree but I would also like it to be more descriptive.
The reason being that with this enum we should be able to add additional sorting options in the future.
ASC_REQUEST_TIME
DESC_REQUEST_TIME

As if we add other options in the future ASC and DESC might be a little generic? ASC on which field etc?